docs/content/connectors/formats/parquet.md

```diff
@@ -0,0 +1,51 @@
+# Protobuf Format
+
+The [Parquet](https://parquet.apache.org/) format allows you to read and write
+Parquet data.
+
+## How to create a table with Parquet format
+
+Here is an example to create a table using the FileSystem connector and Parquet
+format.
+
+If you are using Parquet format in Flink Processor, make sure you have
+configured the environment variable HADOOP_CLASSPATH on both your Flink cluster
+and PyFlink client. Please refer to [Flink's document for Hive
```

Comment:
Yes. If HADOOP_CLASSPATH is not configured, `test_parquet_all_types` would fail at `ExecNodeBase#translateToPlan`, which is executed at the client side.